src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -370,7 +361,7 @@ public override RefKind RefKind
         {
             get
             {
-                return _refKind;
+                return flags.RefKind;
```

Comment:
>return flags.RefKind; [](http://example.com/codeflow?start=16&length=21)

I think the containing property should be moved to the base that declares flags and be "sealed" there.

Reply:
sure.